at-a-glance.rst

```diff
@@ -36,11 +36,10 @@ your overall test suite will become very slow at some point. So, in real
 world we should use both! And that's why you need a **Mink**.
 
 **Mink** removes API differences between different browser emulators providing
-different drivers (read in :doc:`/guides/drivers` chapter) for
-every browser emulator and providing you with the easy way to control the
-browser (:doc:`/guides/session`), traverse pages
-(:doc:`/guides/traversing-pages`) or manipulate page elements
-(:doc:`/guides/manipulating-pages`).
+different drivers (read in :doc:`/guides/drivers` chapter) for every browser
```

Comment:
I'd rather combine links with the text, rather than placing them in parenthesis like so:
```
{different drivers} for every browser emulator and providing you with the easy way to {control the browser}, {traverse pages}, {manipulate page elements} or {interact with them} .
```
The text in `{}` is also a link to corresponding page, but link text is as we want it instead of linked page name.